opentracing_instrumentation/client_hooks/strict_redis.py

```diff
@@ -63,15 +63,38 @@ def peer_tags(self):
     redis.StrictRedis.peer_tags = peer_tags
 
     old_execute_command = redis.StrictRedis.execute_command
+    old_get = redis.StrictRedis.get
+    old_set = redis.StrictRedis.set
+    old_setex = redis.StrictRedis.setex
+
+    def get(self, key):
+        self._extra_tags = [('redis.key', key)]
```

Comment:
I looked at the `StrictRedis` code and it doesn't appear to be threadsafe itself, given that it doesn't use locks. So I don't think it's an issue.